layouts/partials/scripts.html

```diff
@@ -83,4 +83,8 @@
 	}
 
 </script>
-{{ partial "hooks/body-end.html" . }}
\ No newline at end of file
+{{ with .Site.Params.prism_syntax_highlighting }}
```

Comment:
Same, remove this as well. Don't need to include the prism css and js files

Reply:
We do need to include the prism css and js files, as the prism js file is referenced in `layouts/partials/scripts.html` which is actually being over-ridden with the `scripts.html` we have in our jx-docs repository. Then, there's no reference to the prism js file. 

I did cross checked (after your comment) to see if it works or not but it didn't, by excluding the prism js file here.